docs/en/01_Managing_your_website/00_Overview.md

```diff
@@ -11,7 +11,7 @@ The latest SilverStripe CMS administration panel is arranged in four main areas,
 
 This CMS menu allows you to navigate to the main sections of the SilverStripe CMS. The CMS menu can be collapsed by using the toggle at the bottom left of the screen. By default the CMS menu provides you access to your sites pages, files and images, reports, manage security of its members and admins, adjust site settings and CMS User help.
 
-The menu you see may change depending on what modules are installed. However, the following tabs are part of a typical SilverStripe installation:
+The menu you see may change depending on what modules are installed and what permissions your user account has been assigned. The following tabs are part of a typical SilverStripe installation for a user in the "Administrators" group with full  permissions:
```

Comment:
You have an extra space between "full" and "permissions" 🔎

Reply:
Dammit!

I'll fix it